### PR TITLE
HZN-1440: Create tool to audit syslog message parsing

### PR DESCRIPTION
**Pull request by mattixtech**
This PR adds a tool for auditing the syslog messages indexed by the oce-tools project. It will display any messages that failed to parse according to a simple check.

### CODE REVIEW

File: main/pom.xml [hunk 1]
```diff
@@ -116,6 +116,11 @@
             <groupId>args4j</groupId>
             <artifactId>args4j</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.opennms.features.events</groupId>
+            <artifactId>org.opennms.features.events.syslog</artifactId>
+            <version>24.0.0-SNAPSHOT</version>
```
Comment (j-white) [inline main/pom.xml]: This should reference `${opennms.version}` instead.

File: main/src/main/java/org/opennms/oce/tools/main/CpnOnmsSyslogAudit.java [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+    
+    private long processedCount;
+
+    public CpnOnmsSyslogAudit() {
+        super(NAME);
+    }
+
+    @Override
+    public void doExec(Context context) throws Exception {
+        scroll(context.getEsClient().getJestClient(), buildSearch(), EventRecord.class, this::consumeSyslogBatch);
```
Comment (j-white) [inline main/src/main/java/org/opennms/oce/tools/main/CpnOnmsSyslogAudit.java]: Can we use the ESDataProvider#getSyslogRecordsInRange instead? If not, can we at least move this logic to the ESDataProvider class.

**Comment by mattixtech** [comment]
Appears to be working for both header/body cases now. Just need to test it a bit more, found thousands of messages with failed body parsing.

**Comment by mattixtech** [comment]
Tool should be good to go. Used it to fix all the parsing issues I could find.

**Pull request merged**